hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BytesBuffer.java

```diff
@@ -143,14 +151,27 @@ public BytesBuffer write(byte val) {
         return this;
     }
 
+    public BytesBuffer write(int val) {
+        assert val <= UINT8_MAX;
```

Comment:
confused. rename to writeUint8() or writeByte() may be better

Reply:
Since there are many methods need to write a byte of int, such as 0xff, but need to manually convert it like `write((byte) 0xff)`